README.md

```diff
@@ -123,3 +123,8 @@ yarn lint
 - Create new PR with rewarder/\*\* branch name
 - Create newRewarderConfig.json file, fill out value keys from newRewarderConfig.sample.json
 - Once proposed gnosis safe signatures are signed and contracts are verified, we will merge your PR!
+
+## Update Pool allocation
+
+- Create new PR with allocation/\*\* branch name
+- Create allocationConfig.json file, fill out value keys from allocationConfig.sample.json
```

Comment:
Maybe this could be included in the PR, so peeps can just copy the PR and it'll be *chef's kiss* perfect